### ISSUE TITLE
NVDA should be able to change the intonation when reporting formatting changes

### ISSUE THREAD

**Issue by fatihyuksek1** [nvaccess/nvda#11751]
### Describe the solution you'd like
For example, instead of saying underlined when reporting underlined  text, NDVA should be able to either change the pitch or the rate to indicate that there was a formatting change at that part of the text, then when the formatted part ends, it could just revert to normal settings of speech. When I was using narrator for my exams, this feature in narrator helped me a lot as it would change the rate of the speech when the text was highlighted, so I could detect that and I could answer the given questions for that part of the text easily.
